e2e/specs/create.story.spec.js

```diff
@@ -0,0 +1,22 @@
+const SingIn = require('../pages/SignIn');
+let username = 'kevinherrera2';
+let password = '70723844';
+let projectID = '2203114';
+describe('pivotal tracker project page add story', () => {
+    let project;
+    let dashboard;
+    let story;
+
+    before(() => {
+        dashboard = SingIn.loginAs(username, password);
+        project = dashboard.openProjectById(projectID);
+    });
+
+    it('should create a new private project with first account', () => {
+        story = project.getStoryPageOfProject();
+        story.clickAddStoryButton();
+        story.setStoryTitleField("story test");
+        story.clickSaveStoryButton();
+        //expect(`${taskList[0]}`).to.equal(storyTasks.searchTask(taskList[0]));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3ACommentedCode)